testbench/grpc_server.py

```diff
@@ -47,6 +47,11 @@ def CreateBucket(self, request, context):
         self.db.insert_bucket(bucket, context)
         return bucket.metadata
 
+    def GetIamPolicy(self, request, context):
+        self.db.insert_test_bucket()
```

Comment:
Not related to this PR really, but could we factor out this call to happen on server startup maybe?

Reply:
Good idea, will do.